08-Placing-Barriers-in-the-Simulation-Box/03-Wall-geometries.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
<p>'BoxReisize' = typo. Correct to 'BoxResize'.</p><p> </p><p>Also, "... will scale the periodic box and the particle positions but leave all..." should <em>not</em> have a comma before "but" because that is a comma splice; there is no subject in the second clause. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>